site/app/views/grading/ElectronicGraderView.php

```diff
@@ -822,7 +825,11 @@ public function detailsPage(Gradeable $gradeable, $graded_gradeables, $teamless_
             "order_toggle_url" => $details_base_url . '?' .
                 http_build_query(['view' => $view_all ? 'all' : null, 'sort' => $sort === 'random' ? null : 'random', 'anon_mode' => $anon_mode]),
             "sort" => $sort,
-            "direction" => $direction
+            "direction" => $direction,
+            "csrf_token" => $this->core->getCsrfToken(),
```

Comment:
The csrf token is already brought into the output. You can access this in Javascript by calling csrfToken since it is brought in from server.js.